### PR TITLE
Fix setSettings on reindex

### LINKED ISSUE #51: $algoliaSettings['attributesForFaceting'] setting is being ignored.
Hi there,

Each time I run `Product::reindex();` the `attributesForFaceting` setting gets cleared on my dashboard and I must login and update it manually. Am I doing it wrong?

```
class Product extends \ResourceModel {
	use AlgoliaEloquentTrait;
	public static $perEnvironment = true;
	public static $autoIndex = true;
	public $algoliaSettings = [
		'attributesForFaceting' => [ 'category_id', 'vendor_id', 'unit_price', 'brand' ],
	];
}
```

Thanks in advance!

### PR DESCRIPTION
**Pull request by JanPetr**
Should fix #51. Review please, not sure I covered all possibilities.

### CODE REVIEW

File: src/AlgoliaEloquentTrait.php [hunk 1]
```diff
@@ -15,15 +15,25 @@
...
         $modelHelper = App::make('\AlgoliaSearch\Laravel\ModelHelper');
 
         $indices = $modelHelper->getIndices($this);
         $indicesTmp = $safe ? $modelHelper->getIndicesTmp($this) : $indices;
 
+        if ($setSettings === true) {
+            $setToTmpIndices = false;
+            if ($safe === true) {
+                $setToTmpIndices = true;
+            }
```
Comment (maxiloc) [inline src/AlgoliaEloquentTrait.php]: $setToTmpIndices = ($safe === true);
might be a bit easier to read

**Comment by maxiloc** [comment]
Good 👍

**Comment by maxiloc** [comment]
LGTM

File: src/AlgoliaEloquentTrait.php (hunk 1 above)
Comment (JanPetr) [inline src/AlgoliaEloquentTrait.php]: Fixed

**Pull request merged**